os/android/utils_android.h

```diff
@@ -58,6 +58,8 @@ static uint32_t GetDrmFormatFromHALFormat(int format) {
       return DRM_FORMAT_ARGB8888;
     case HAL_PIXEL_FORMAT_YV12:
       return DRM_FORMAT_YVU420;
+    case HAL_PIXEL_FORMAT_RGBA_FP16:
```

Comment:
Can you remove the OAM jira and in tests add what test starts passing with this change ?

Reply:
Okay.